website/hw/hw1_rubric.rst

```diff
@@ -0,0 +1,74 @@
+Homework 1 Rubric
+=================
+
+The grading of this project follows a specifications grading approach. If you have not already
+done so, make sure to read our `Grading <../grading.html>`__ page for more details.
+
+When grading your work, we will be checking for completeness (i.e., how much of the
+homework you completed) and for quality (whether you completed the exercises in
+the homework correctly).
+
+Completeness
+------------
+
+The graders will be checking your GitHub repository to verify whether you completed the steps
+described in Parts I and Parts II of the homework. They will expect your commits to look roughly like this:
+
+**Part I**
+
+- ``first commit``
+- ``Updated README.md``
+- ``Homework 1 updates:``
+
+  ``- Added test.txt``
+
+  ``- Updated README.md file``
+
+- ``A few more changes``
+- ``Adding one more change``
+- ``Our last change?``
+- ``Wrapping up first part of the homework``
+
+**Part II**
+
+- ``Added echo.c``
+- ``Updated author in echo.c``
+- ``Updated buffer size to 1000``
+- ``Updated buffer size to 250``
+- ``Merge branch 'update-buffer-size' into main``
+
+Please note that this is not a strict list of commits: if you made a mistake at some point,
+and had to create additional commit messages because of it, you will not be penalized as long
+as it's clear that you were, overall, following the instructions in Parts I and II. On the other hand,
+if your list of commits looks like the ones above, that's a pretty good indicator that you're
+in good shape (note: the graders will still be looking at the contents of the commits to
+verify that you actually performed the steps in the homework, as opposed to just creating a bunch
+of commits with the expected titles)
+
+The graders will also check whether you've provided answers for the exercises in Part III.
+
+You will earn, at least, the following ESNU score depending on how much work you completed:
+
+- Satisfactory: Followed the instructions for Parts I and II mostly correctly, and submitted work for Part III
```

Comment:
I think it might be good to specify here, that the rubric to attain an excellent is covered in the "quality" section of the rubric. I had falsely assumed that the completeness and quality grade were two separate scores, while this rubric seems to imply that it is just a singular score?

Reply:
I've tweaked the language to make this clearer.